### DISCUSSION TITLE
Power

### DISCUSSION THREAD

**Discussion by Bigred200** [riffnshred/nhl-led-scoreboard#391] [Hardware]
So i thought i had a defective pi 4b but it is possible that i fried it. Couple questions. i have a 3(b maybe but i cant remember a buddy is giving it to me tomorrow so ill know then) --- i saw you said in a previous post with the 3 you could power just the hat and if u didnt have anything plugged in it would be fine. is that powering the Matrix from the bonnet as well? so all three powered by the same supply? - Can i power the pi by itself and then also power the bonnet thats powering the matrix just to stay safe? --- My issues with my four occurred shortly after activating the pwm. I ran a reboot and it was fine but then next time i started it. it never booted. pi was dead. Any insight on that? I did the mod correctly and my soldering looked great no shorts. both the pi and bonnet were being powered. It was running fine for approx 3 days worth of testing before it stopped

**Comment by riffnshred** [comment]
You definitly can power a Pi 3 by using the same power source the HAT/Bonnet is connected to. The latest can feed up to 1 Amp of current which is just enough to feed a Pi 3. The Pi 4 on the other hand need more then 1 Amp to function under heavy use and if there is not enough, it can cause issues and might even kill it.

Lets say you had the Pi 4 use its own power source, the other thing that might have killed it is if anything on the Pi was touching the bottom of the HAT of Bonnet. This is the most common reason why some people kill their raspberry pi. Otherwise it might be related to your soldering job on the bonnet.  Its actually hard to kill a Pi in any other way. Either its undercurrent/undervoltage or its a short. 

To be safe, I recommand puting capton tape of electrical tape underneath my Bonnets.

**Comment by riffnshred** [comment]
To answer your questions:

The Matrix is powered stricly by the Bonnet/HAT. Adafruit specificly say that the Bonnet can power the Pi but is not meant to be this way and they added a 1Amp limiter to protect the Pi (Im not sure from what).

You can use a seperate PSU for your Pi (it is recommanded by adafruit). I never had too in 4 years of using those and I never had an issue with Pi3 or Pi Zero 2 W.

**Comment by riffnshred** [comment]
Also, might be less obvious for some but, have you tried a new SD card with your Pi 4 ? Might just have killed your SD card and the Pi 4 is fine.

**Reply by Bigred200** [reply]
Yeah unfortunately I tried several different sd, booting from USB, reflashing. Green and red light stay solid no matter what. I think that must have been what happened was they touched. It happened when I booted after putting everything in the case. Just didn't think I applied that much pressure to them. Printed the case specifically so it could all breathe and have plenty of space. Oh well. When I get the 3 tomorrow I'll tape it and hope for the best. Probably going to power both for at least the time it takes to get running and then I suppose we will go from there. Thank you!
